src/main/java/no/nav/pto/veilarbportefolje/aktiviteter/AktivitetStatusRepositoryV2.java

```diff
@@ -0,0 +1,56 @@
+package no.nav.pto.veilarbportefolje.aktiviteter;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.pto.veilarbportefolje.database.PostgresTable;
+import no.nav.pto.veilarbportefolje.domene.Brukerdata;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import static no.nav.pto.veilarbportefolje.database.PostgresTable.AKTIVITETTYPE_STATUS.*;
```

Comment:
Kanskje endre import til følgende, siden det er queries på for forskjellige tabeller i filen
```
import static no.nav.pto.veilarbportefolje.database.PostgresTable.AKTIVITETTYPE_STATUS;
import static no.nav.pto.veilarbportefolje.database.PostgresTable.AKTIVITET_STATUS;
```